pkg/metrics/metrics.go

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```

Comment:
I see that in some source files we are adding a new line immediately before the `package` declaration and in some files not (example `aws_secrets.go`) why is that?

Reply:
There is always a line after `// limitations under the License.`
incase of `aws_secrets.go`, there a package description ` Package repositoryserver validates the secret credentials for all location profiles.` written before `package` (for style check linter ST1000) which is a different issue to be solved in another PR(as I am not aware of all the package descriptions)

Reply:
ok